types/ioredis/ioredis-tests.ts

```diff
@@ -177,6 +177,8 @@ new Redis.Cluster([{
 
 redis.xack('streamName', 'groupName', 'id');
 redis.xadd('streamName', '*', 'field', 'name');
+redis.xadd('streamName', 'MAXLEN', 100, '*', 'field', 'name');
```

Comment:
Might be nice to test `'maxlen'` as well.

Reply:
Good point, I'll create another PR